### DISCUSSION TITLE
Change IP of cameras

### DISCUSSION THREAD

**Discussion by UVivarelli** [fwestenberg/reolink_dev#524] [Q&A]
After a reboot of my router, IP of cameras has changed. I've tried to find the IP address of cameras on my HA configuration, but without success!
Where can I find it to change? If is it possible I don't want unistallall and re-install the cameras.....

**Comment by FredrikNas** [comment]
In your router! 

I'd advice you to lock the IP for each camera so that they dont change when you reboot your equipment. This can also be done in your router.
